_content/monographs/Kamma-and-its-fruit_nyanaponika.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Kamma and Its Fruit"
+editor:
+  - nyanaponika
+external_url: "https://www.bps.lk/olib/bp/bp413s_Nyanaponika_Kamma-And-Its-Fruit-Essays.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1sWvzdHq7rhXBdvCZ0sRqrOiDXHFByptH/view?usp=drive_link
+  - "https://drive.google.com/file/d/1sXAdAYGRT6XemHGqWXXC4KkIEfV7dmaH/view?usp=drive_link
+course: karma
+tags:
+  - ethics
+year: 1975
+olid: OL13056102M
+oclc: 313690068
+publisher: bps
+pages: 96
+---
+
+> Kamma is action; vipáka is result. Therefore kamma is the active principle; vipáka is the passive mode of coming-to-be.
+
+A collection of essays by five different authors that explain actions and results and how this understanding plays a role in cultivating the Buddha path.
```

Comment:
Can we add the table of contents from the back cover?

```markdown
Containing:
1. Action — Francis Story
2. Kamma and Causality — Francis Story
3. ...
```
That way if people are searching for a particular essay they will find it 😁